gokart/run.py

```
@@ -12,8 +13,39 @@
 import gokart.slack
 from gokart.object_storage import ObjectStorage
 
+from opentelemetry import trace
+from opentelemetry.exporter.cloud_trace import CloudTraceSpanExporter
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import BatchSpanProcessor
+from opentelemetry.sdk.trace.export import ConsoleSpanExporter
+
 logger = getLogger(__name__)
 
+tracer_provider = TracerProvider()
+try:
+    cloud_trace_exporter = CloudTraceSpanExporter()
+    tracer_provider.add_span_processor(
+        BatchSpanProcessor(cloud_trace_exporter)
+    )
+except:
+    traceback.print_exc()
```

temporary suppressing exception since now always initializing. otherwise unit tests will fail.